frontend/src/scss/global.scss

```diff
@@ -98,3 +98,11 @@ body {
 .d-flow-root {
   display: flow-root;
 }
+
+.d-contents {
+  display: contents;
+}
+
+.currentColor--text {
+  color: currentColor !important;
+}
```

Comment:
I don't see this being used either..?

Reply:
Yes it is. Vuetify automatically constructs these if you provide `color="currentColor"`